up-raspberry/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+
+services:
+  mm_raspberry:
+    container_name: mm
+    image: bastilimbach/docker-magicmirror:latest
+    ports:
+      - "8080:8080"
+    volumes:
+      - ~/magic_mirror/config:/opt/magic_mirror/config
+      - ~/magic_mirror/modules:/opt/magic_mirror/modules
+      - /tmp/.X11-unix:/tmp/.X11-unix
+#1      - /opt/vc:/opt/vc/:ro
+#3    devices:
+#3      - /dev/vchiq
+    environment:
+#1      LD_LIBRARY_PATH: /opt/vc/lib
+      DISPLAY: unix$DISPLAY
+#2    network_mode: host
+    entrypoint: /opt/magic_mirror/docker-entrypoint.sh "npm start"   
+    restart: always
+
+#1  needed for module MMM-Systemtemperature
+#2  needed for module MMM-NetworkScanner
+#3  needed for #1 and #2
```

Comment:
With more 3rd party modules, I think this lists will get bigger and bigger. Look at the above comment.